circle.yml

```diff
@@ -0,0 +1,12 @@
+dependencies:
+  override:
+    - printf '%s\n' color=false progress=false >~/.npmrc
+    - make setup
+
+machine:
+  node:
+    version: 4
```

Comment:
> Where is the server?

On Linode currently.

> It would be nice to do it with infrastructure as code, maybe use a Docker container so its configuration would be in source control.

Sounds great! This needn't happen as part of this initial pull request though. ;)